packages/webapp/src/pages/Logs/Search.tsx

```diff
@@ -181,27 +189,44 @@ export const LogsSearch: React.FC = () => {
     );
     useEffect(
         function syncStateToQueryParams() {
-            // reset pagination and stored items
+            if (!synced) {
+                return;
+            }
 
             // Sync the state back to the URL for sharing
-            const tmp = new URLSearchParams({
-                states: states as any,
-                integrations: integrations as any,
-                connections: connections as any,
-                syncs: syncs as any,
-                types: types as any,
-                live: String(isLive)
-            });
+            const tmp = new URLSearchParams();
+            if (states.length > 0 && JSON.stringify(states) !== JSON.stringify(statusDefaultOptions)) {
+                tmp.set('states', states as any);
+            }
+            if (integrations.length > 0 && JSON.stringify(integrations) !== JSON.stringify(integrationsDefaultOptions)) {
+                tmp.set('integrations', integrations as any);
+            }
+            if (connections.length > 0 && JSON.stringify(connections) !== JSON.stringify(connectionsDefaultOptions)) {
+                tmp.set('connections', connections as any);
+            }
+            if (syncs.length > 0 && JSON.stringify(syncs) !== JSON.stringify(syncsDefaultOptions)) {
+                tmp.set('syncs', syncs as any);
+            }
+            if (types.length > 0 && JSON.stringify(types) !== JSON.stringify(typesDefaultOptions)) {
```

Comment:
does the string comparison only work if the param arrays are sorted the same way?

Reply:
good point, will change that